.github/bin/retry

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Please specifiy in commit message header that this is about GH actions.

Reply:
Modified commit message to "Add retry logic in Github Actions"